server.key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This file should not be added to the repo.

Reply:
Done.